views/homework/extended.hbs

```diff
@@ -24,21 +24,21 @@
 			{{#if isTeacher}}
 				{{#userHasPermission "HOMEWORK_EDIT"}}
 					<a class="btn btn-primary btn-edit" href="/homework/{{../_id}}/edit?returnUrl={{../_id}}"
-						alt="{{$t "homework._task.label.editTaskAlt" (dict "title" ../title)}}"> <i class='fa fa-edit'></i> {{$t "global.button.edit" }}
+						alt="{{$t "homework._task.label.editTaskAlt" (dict "title" ../title)}}" data-testid="task-details-btn-edit"> <i class='fa fa-edit'></i> {{$t "global.button.edit" }}
 					</a>
 					<a class="btn btn-secondary btn-delete" href="/homework/{{../_id}}" redirect="/tasks" target="_blank"
-						data-method="DELETE" data-name="{{../title}}" alt="{{$t "homework._task.label.deleteTaskAlt" (dict "title" ../title)}}">
+						data-method="DELETE" data-name="{{../title}}" alt="{{$t "homework._task.label.deleteTaskAlt" (dict "title" ../title)}}" data-testid="task-details-btn-delete">
 						<i class='fa fa-trash-o'></i> {{$t "global.headline.delete" }}
 					</a>
 					<a class="btn btn-secondary btn-copy" data-toggle="tooltip" title="{{$t "homework._task.label.copyTask"}}" href="{{@root.copyServiceUrl}}"
-						alt="{{$t "homework._task.label.copyTaskAlt" (dict "title" ../title)}}"> <i class='fa fa-copy'></i> {{$t "homework.button.copy" }} </a>
+						alt="{{$t "homework._task.label.copyTaskAlt" (dict "title" ../title)}}" data-testid="task-details-btn-copy"> <i class='fa fa-copy'></i> {{$t "homework.button.copy" }} </a>
 				{{/userHasPermission}}
 			{{/if}}
 
 			{{#inArray this.currentUser._id this.archived}}
 				{{#userHasPermission "HOMEWORK_CREATE"}}
 				<a class="btn btn-secondary btn-archive" href="{{this.url}}" target="_blank" data="archive=open"
-					data-toggle="tooltip" title="{{$t "homework._task.label.moveFromArchive"}}"
+					data-toggle="tooltip" title="{{$t "homework._task.label.moveFromArchive"}}" data-testid="task-details-btn-restore"
```

Comment:
please put the attribute behind `alt-text` to be consistent with the other orders

Reply:
done